### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

**Comment by agramfort** [comment]
all green @qbarthelemy !

File: pyriemann/__init__.py [hunk 1]
```diff
@@ -8,3 +8,14 @@
 from . import embedding
 
 __version__ = "0.2.7.dev"
+
+__all__ = [
+    'classification',
```
Comment (qbarthelemy) [inline pyriemann/__init__.py:13]: We must add the new module `preprocessing`.

File: pyriemann/utils/mean.py [hunk 2]
```diff
@@ -181,12 +185,12 @@ def mean_logdet(covmats, tol=10e-5, maxiter=50, init=None, sample_weight=None):
 
 def mean_wasserstein(covmats, tol=10e-4, maxiter=50, init=None,
                      sample_weight=None):
-    """Return the mean covariance matrix according to the wasserstein metric.
+    r"""Return the mean covariance matrix according to the wasserstein metric.
```
Comment (qbarthelemy) [inline pyriemann/utils/mean.py:188]:
```suggestion
    r"""Return the mean covariance matrix according to the Wasserstein metric.
```

File: pyriemann/utils/geodesic.py [hunk 3]
```diff
@@ -65,10 +67,11 @@ def geodesic_euclid(A, B, alpha=0.5):
 
 
 def geodesic_logeuclid(A, B, alpha=0.5):
-    """Return the matrix at the position alpha on the log euclidean geodesic between A and B  :
+    r"""Return the matrix at the position alpha on the log euclidean geodesic
```
Comment (qbarthelemy) [inline pyriemann/utils/geodesic.py:70]:
```suggestion
    r"""Return the matrix at the position alpha on the log-Euclidean geodesic
```

File: pyriemann/utils/geodesic.py [hunk 4]
```diff
@@ -48,10 +49,11 @@ def geodesic_riemann(A, B, alpha=0.5):
 
 
 def geodesic_euclid(A, B, alpha=0.5):
-    """Return the matrix at the position alpha on the euclidean geodesic between A and B  :
+    r"""Return the matrix at the position alpha on the euclidean geodesic
```
Comment (qbarthelemy) [inline pyriemann/utils/geodesic.py:52]:
```suggestion
    r"""Return the matrix at the position alpha on the Euclidean geodesic
```

File: pyriemann/utils/geodesic.py [hunk 5]
```diff
@@ -26,10 +26,11 @@ def geodesic(A, B, alpha, metric='riemann'):
 
 
 def geodesic_riemann(A, B, alpha=0.5):
-    """Return the matrix at the position alpha on the riemannian geodesic between A and B  :
+    r"""Return the matrix at the position alpha on the riemannian geodesic
```
Comment (qbarthelemy) [inline pyriemann/utils/geodesic.py:29]:
```suggestion
    r"""Return the matrix at the position alpha on the Riemannian geodesic
```

File: tests/test_utils_covariance.py [hunk 6]
```diff
@@ -8,31 +8,36 @@
                                         normalize, get_nondiag_weight)
 
 
-def test_covariances():
+@pytest.mark.parametrize(
```
Comment (qbarthelemy) [inline tests/test_utils_covariance.py:11]: 👍 
The beginning of a new world...

**Comment by agramfort** [comment]
thx @qbarthelemy for the review !

**Pull request merged**